mods/default/functions.lua

```diff
@@ -427,7 +427,7 @@ local function leafdecay_after_destruct(pos, oldnode, def)
 			vector.add(pos, def.radius), def.leaves)) do
 		local node = minetest.get_node(v)
 		local timer = minetest.get_node_timer(v)
-		if node.param2 == 0 and not timer:is_started() then
+		if node.param2 ~=1 and not timer:is_started() then -- Use ~=1 to make the other value available (fix luscious mod)
```

Comment:
Please remove '(fix luscious mod)' as the fix is universally useful and we don't mention non-MTG mods in comments.

Reply:
Actually, the whole comment is not necessary.